force use of -O1 when compiling colCounts.c
workaround for "internal compiler error" when building with Intel compilers on an Intel Skylake system
author: B. Hajgato (Free University Brussels - VUB)
--- matrixStats.org/MD5 2018-02-11 15:13:37.000000000 +0100
+++ matrixStats/MD5     2018-04-23 21:56:14.221226594 +0200
@@ -278,6 +278,7 @@
 536a4eddd5d389a2f37cac3702381be7 *src/x_OP_y.c
 5227137bc874290ef86b5d1b7e2878c2 *src/x_OP_y_lowlevel.h
 ded428cafc1ea29e648f78a70297e459 *src/x_OP_y_lowlevel_template.h
+6d47ab21f006d1c9ed1582778c0d7f78 *src/Makevars
 8a7346f77a61c642a27791fa78ff9bb8 *tests/allocArray.R
 d7014909df5dcc968e46e858a8c2686b *tests/allocMatrix.R
 508a5d07b8897d7c461cd73559a0c22f *tests/allocVector.R
--- matrixStats.org/src/Makevars        1970-01-01 01:00:00.000000000 +0100
+++ matrixStats/src/Makevars    2018-04-23 21:55:46.953254629 +0200
@@ -0,0 +1,4 @@
+all: $(SHLIB)
+
+colCounts.o : CFLAGS+=-O1
+
